### PR TITLE
Refactor Digital Media page

### PR DESCRIPTION
**Pull request by TomDijkema**
PR refactors the Digital Media page basics and implements the absolute basics of the page: showing the top bar, ID card, image viewer and digital media list. Digital Media annotations will follow up.

### CODE REVIEW

**Comment by TomDijkema** [comment]
Not going to fix these Sonar duplication issues as I want to keep these as separate files.

File: src/api/digitalMedia/GetDigitalMedia.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * @returns Object of Digital Media
+ */
+const GetDigitalMedia = async({ handle, version } : { handle: string, version?: number }) => {
+    let digitalMedia: DigitalMedia | undefined;
+
+    if (handle) {
+        let endPoint: string;
+
+        if (version) {
+            endPoint = `digital-media${handle}/${version}`;
```
Comment (southeo) [inline src/api/digitalMedia/GetDigitalMedia.ts:25]: missing a slash between digital media and $handle ->  digital-media/${handle}/${version}

File: src/api/digitalMedia/ScheduleDigitalMediaMAS.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const ScheduleDigitalMediaMAS = async ({ handle, masList }: { handle: string, masList: { masId: string }[] }) => {
+    let digitalMediaMAS: Dict = {};
+
+    if (handle) {
+        const token = KeycloakService.GetToken();
+
+        const masRecord: {
+            data: {
+                type: 'MasRequest',
+                attributes: {
```
Comment (southeo) [inline src/api/digitalMedia/ScheduleDigitalMediaMAS.ts:27]: i think this is meant to be: 
```
attributes: {
            mass: [
                {
                    masId: string
                }
            ]
}
```

**Review by southeo — APPROVED** [review]
2 things to fix first but here's an approval bc you deserve it tommo

File: src/api/digitalMedia/ScheduleDigitalMediaMAS.ts (hunk 2 above)
Comment (TomDijkema) [reply src/api/digitalMedia/ScheduleDigitalMediaMAS.ts:27]: This is the typing and when you want to indicate a type is an array in TypeScript, you put the block brackets after the value, for example, an array containing strings would be: string[], or an array of objects holding a name: {name: hello}[]

**Pull request merged**